com.locomalito.abbayedesmorts.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Abbaye des Morts
+Comment=Indie platform game by Locomalito
+Comment[ca]=Joc indie de plataformes de Locomalito
+Comment[es]=Juego indie de plataformas de Locomalito
+Exec=abbayev2
+Icon=abbaye
```

Comment:
```suggestion
Icon=com.locomalito.abbayedesmorts
```

Reply:
You will need to install the icon with the correct name too.

Reply:
fixed in 0cc2245e0ae4d3ce05733da505675804c9915d74